mycpp/mark_sweep_heap.h

```diff
@@ -265,10 +265,12 @@ class MarkSweepHeap {
 #ifndef NO_POOL_ALLOC
   // 16,384 / 24 bytes = 682 cells (rounded), 16,368 bytes
   // 16,384 / 48 bytes = 341 cells (rounded), 16,368 bytes
+  // 16,384 / 96 bytes = 171 cells (rounded), 16,368 bytes
```

Comment:
Hm this still says 96

I kinda want to do some evaluation, make sure it's not over-tuned for Linux / glibc / 64-bit

And does this waste memory on any workloads?

Reply:
Also I think we had the theory again that malloc() needed some slack, to be strictly less than 16,384 bytes, to allow for headers

But I guess we never measured that effect

We already run on many different libc with different malloc(), so yeah I think we can at least survey Alpine Linux

Reply:
Oh whoops. That's a typo leftover from a previous version. Will fix.

Happy to leave this open until we have a better idea of the effect with different allocators. I suspect that it will be a win in most cases, though. Since the write to the allocation header (whether it's below the buffer or elsewhere) is going to cause page faults (the reduction of which was the main win from this) regardless.